pkgs/tools/security/otpgen/default.nix

```diff
@@ -0,0 +1,42 @@
+{ 
+  pkgs ? import <nixpkgs> {}
```

Comment:
Do not import nixpkgs here. This will be provided from outside. The argument list needs a through clearing. Please have a look at other default.nix files in nixpkgs.

Reply:
Fixed here https://github.com/NixOS/nixpkgs/commit/5664fd4112b3dfedc09e489d46a269f97d566a61